src/arrays/Arrays.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace arrays;
+
+class Arrays implements ArraysInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function repeatArrayValues(array $input): array
+    {
+        $new_array = [];
```

Comment:
please use camelCase names

Reply:
> please use camelCase names

All changes done. Now I'm proud of my code here ;)